stories/topics/proc-set-monitoring-logging-yml.adoc

```diff
@@ -0,0 +1,22 @@
+[id="proc-gcp-set-monitoring-logging-yml"]
+
+= Updating the data file
+
+If you do not require a parameter, remove that parameter from the configuration file.
+
+.Procedure
+. Edit the `/tmp/logging-monitoring.yml` file and set the following parameters:
+
+* `ansible_config_path` is used by default as the standard configuration for the `ansible-on-cloud offering` but if you have extra requirements in your environment you can specify your own. 
+* `cloud_credentials_path` is the absolute path toward your credentials. 
+This must be an absolute path.
+* `deployment_name` is the name of the deployment.
+* `components` is an array containing the type of component where you want to do the setup. 
+The default is [ “controller”, “hub” ] which means that the logging monitoring will be enabled on both {controllerName} and {HubName}.
+* `monitoring_enabled` is set to `true` to enable the monitoring, `false` otherwise. Default = `false`.
+* `logging_enabled` is set to `true` to enable the logging, `false` otherwise. Default = `false`.
+* `default_collector_interval` is the periodicity at which the monitoring data must be send to the Google Cloud. 
+Default = 59s. 
+The Google cost of this service depends on that periodicity and so the higher the value of the  collector interval, the less it will cost. 
```

Comment:
Wondering if we want to somehow make the Google cost part stand out a little more with **BOLD** or a NOTE or some other formatting to make it more visible